### ISSUE TITLE
Special chars in keys not supported

### ISSUE THREAD

**Issue by bend-n** [Canop/deser-hjson#9]
```
@?;'"\/.: value
```
```rs
Syntax { line: 2, col: 7, code: ExpectedMapColon, at: "\\/.: value\n}\n" }
```
<sup>sorry for making so many issues btw</sup>

**Comment by Canop** [comment]
Hum... it looks like I didn't see this issue...

The problem is with the `"`, not the special chars.

<sup>sorry too</sup>

**Comment by Canop** [comment]
There doesn't seem to be any problem with accepting a quote in a "quoteless" key: https://github.com/Canop/deser-hjson/pull/12

**Comment by Canop** [comment]
Fix released in 1.1.1
